sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/ComponentSamples.cs

```diff
@@ -63,8 +63,7 @@ public async Task RunSamplesAsync(DigitalTwinsClient client)
                         "Component1",
                         new ModelProperties
                         {
-                            // model Id of component
-                            Metadata = { ModelId = componentModelId },
+                            Metadata = { },
```

Comment:
Not necessary

Reply:
ADT service is expecting Metadata property in the payload, even if it's empty. I tried removing it from the payload and the services rejects it.

Reply:
The object is already instantiated by default. This would just initialize additional properties. I'm confused as to why including this would ensure it was serialized or not.

Reply:
Fixed with David's help. Sending a PR for this soon.

Reply:
addressed this comment with this PR:
https://github.com/Azure/azure-sdk-for-net/pull/12606